packages-proprietary/tokens/src/components/ams/page-header.tokens.json

```diff
@@ -24,6 +24,7 @@
         "text-wrap": { "value": "{ams.typography.heading.text-wrap}" }
       },
       "mega-menu": {
+        "padding-block": { "value": "{ams.space.xl}" },
```

Comment:
At the top, the padding gets complemented with the row gap of `.ams-page-header__navigation` which can be seen as a bug. That gap is intended to separate the logo from a wrapping header menu, but it also adds space between said menu and the mega menu.